src/seckrit.py

```diff
@@ -10,11 +10,22 @@
 
 
 def main(args):
+
     # Load the YAML file that tells us which secrets we want to load (and what to do with them).
-    manifest = load_manifest(args.manifest_file)
+    if args.manifest:
+        manifest = load_manifest(args.manifest)
+    elif "SECKRIT_MANIFEST" in os.environ:
+        manifest = yaml.safe_load(os.environ.get('SECKRIT_MANIFEST'))
+    else:
+        raise RuntimeError("Should pass --manifest argument or define SECKRIT_MANIFEST environment variable")
```

Comment:
This is a user error, so it shouldn't crash out to the console with a stacktrace. Just `print()` the message and `exit(1)`.